apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -51,6 +51,21 @@ const show = computed(() => {
   return (componentName: (typeof COMPONENTS)[number]) => activeConfig.value.includes(componentName);
 });
 
+const DENSITIES = ["Compact", "Default", "Cozy"];
+type DensityType = "default" | "cozy" | "compact" | undefined;
```

Comment:
We already have this inside /composables/density.ts, we just need to export it :)